docs/connectors/building-connectors/using-a-custom-connector.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Using a Custom Connector"
+sidebar_position: 5
+---
+
+## Adding the Connector to Conduit
+
+The Conduit Connector template includes a `Makefile` that can be used to generate a binary version of your connector
+
+```bash
+make build
+```
+
+After the command runs you should see a binary file with the name of the connector in the local directory.
+
+To run the Connector, you will need to put it in a directory that Conduit is able to access.  By default, this directory is the `connectors` directory located alongside the Conduit binary.  Move your newly created Connector binary into the `connectors` directory and run Conduit using the following command:
+
+```bash
+./conduit
+```
+
+Navigate to the Conduit UI (by default located at [http://localhost:8080/ui](http://localhost:8080/ui) while Conduit is running) to see your Connector in the list of available Connectors
+
+![Custom Connector in Conduit UI](/img/conduit-custom-connector-ui-screenshot.png)
```

Comment:
noticed we still have the other images with code snippets and they're not used anymore 👀

Reply:
@William-Hill No you got them all! you just forgot to delete the other screenshots from the images folder :D